frappe/desk/doctype/notification_log/notification_log.py

```diff
@@ -112,11 +113,25 @@ def get_title_html(title):
 	return '<b class="subject-title">{0}</b>'.format(title)
 
 @frappe.whitelist()
-def mark_as_seen(docname):
-	if docname:
-		frappe.db.set_value('Notification Log', docname, 'seen', 1, update_modified=False)
+def mark_all_as_read(docnames):
+	docnames = frappe.parse_json(docnames)
+	if docnames:
+		filters = {'name': ['in', docnames]}
+		frappe.db.set_value('Notification Log', filters, 'read', 1, update_modified=False)
 
+@frappe.whitelist()
+def mark_as_read(docname):
+	if docname:
+		frappe.db.set_value('Notification Log', docname, 'read', 1, update_modified=False)
 
 @frappe.whitelist()
 def trigger_indicator_hide():
 	frappe.publish_realtime('indicator_hide', user=frappe.session.user)
+
+def set_notifications_as_unseen(user):
+	try:
+		notification_settings_doc = frappe.get_doc('Notification Settings', user)
+		if notification_settings_doc.seen == 1:
```

Comment:
No need to check if it is `1`, just set it anyway. This will also get rid of `get_doc`.